src/MusicStore/Controllers/ManageController.cs

```diff
@@ -119,25 +116,25 @@ public IActionResult AddPhoneNumber()
         // POST: /Manage/DisableTwoFactorAuthentication
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> DisableTwoFactorAuthentication()
+        public async Task<IActionResult> DisableTwoFactorAuthentication(CancellationToken cancellationToken)
```

Comment:
Can we rename the variable name to `requestAborted` to indicate what this token is for?